buda/entities/payment.proto

```diff
@@ -24,5 +24,5 @@ message Payment {
     PaymentType payment_type = 8;
     string payment_amount = 9;
     string payment_currency = 10;
-    int32 subscription_payment_sequence = 11;
+    PaymentTermsAndSchedule payment_terms_and_schedule = 11;
```

Comment:
ah yeah, good one!  I meant to actually remove this one, as I think it maybe isn't the best fit in `payment` anymore